### DISCUSSION TITLE
Max invite expiry is 24 hours!

### DISCUSSION THREAD

**Discussion by cliveportman** [supabase/supabase#8647] [Questions] [answered]
It looks like the link in an invite email expires after 24 hours. I've found the field in settings to change it, but that's already the max. This is going to cause my clients the world of pain, because there's no way most of their users will click the link within 24 hours. Can this max be overridden at all?

**Comment by GaryAustin1** [comment, marked as answer]
Does not appear you can make it longer.
https://github.com/supabase/supabase/discussions/6603
That also suggests a HACK in that you might be able to use cron to scan the auth.users table and add time by adjusting the date the token was "issued".
